chore(colors): remove duplicate colors

jniles opened the pull request:
This commit removes duplicate colors from the ColorService.

Closes #7018

jniles committed:
chore(colors): remove duplicate colors

This commit removes duplicate colors from the ColorService.

Closes #7018

chore(colors): adjust casing of colors

Removes the capitalization of letters in the color service.

jmcameron commented on client/src/js/services/color.js:
```diff
@@ -6,17 +6,6 @@ ColorService.$inject = [
 function ColorService($translate) {
   const service = this;
   service.list = [
-    { name : $translate.instant('COLORS.AQUA'), value : '#00ffff' },
```
Thank you!

jmcameron reviewed (changes requested):
I noticed this problem while converting the E2E tests.   Thanks for working on it!

The changes mostly look good.  

I tested it using the bhima_test DB.  I noticed that the colors that are from i18n COLORS.* are capitalized.  Could you please modify the .json file and uncapitalize them.  Also I think GRAY is a duplicate that could be deleted.

Thanks!

jmcameron reviewed (changes requested):
Thanks for the changes.  I hate to be picky, but could I ask you to delete the duplicate 'gray' color and sort the list of colors alphabetically.  And why do we have translations for some colors and not others (legacy?).

jniles committed:
chore(colors): add additional colors, rm dupes

This commit removes duplicates in the color list, ensures it is sorted,
and adds the full additional colors to the color service.
